Casks/r/resolume-arena.rb

```diff
@@ -2,20 +2,26 @@
   version "7.21.3,38686"
   sha256 "6bfd580965c1cd296cfe6eb179f1558096b9dc83d18e138612278e2eb943e6f9"
 
-  url "https://dd5sgwxv3xok.cloudfront.net/Resolume_Arena_#{version.major_minor_patch.dots_to_underscores}_rev_#{version.csv.second}_Installer.dmg",
+  url "https://dd5sgwxv3xok.cloudfront.net/Resolume_Arena_#{version.csv.first.dots_to_underscores}_rev_#{version.csv.second}_Installer.dmg",
       verified: "dd5sgwxv3xok.cloudfront.net/"
   name "Resolume Arena"
   desc "Video mapping software"
   homepage "https://resolume.com/"
 
   livecheck do
-    url "https://resolume.com/download/"
-    strategy :page_match do |page|
-      page.scan(/href=.*?Arena[._-]v?(\d+(?:[._-]\d+)+)[._-]rev[._-](\d+).+\.dmg/i)
-          .map { |match| "#{match[0].tr("_", ".")},#{match[1]}" }
+    url "https://resolume.com/update/arena_updates_mac.xml"
+    regex(/^v?(\d+(?:\.\d+)+).*?(\d+)$/i)
```

Comment:
```suggestion
    regex(/^v?(\d+(?:\.\d+)+)\s*rev\s*(\d+)$/i)
```

Using `.*?` here can lead to unexpected matches, so it may be better to explicitly match `rev` (i.e., potentially better to fail with an error if the format changes instead of surfacing an incorrectly-parsed version).